### ISSUE TITLE
 [moduleId].call() is not a function.

### ISSUE THREAD

**Issue by shanqq** [axios/axios#6532]
### Describe the bug

This is the line thats getting failed.
var _utils_js__WEBPACK_IMPORTED_MODULE_2__ = __webpack_require__(/*! ./utils.js */ "./node_modules/axios/lib/utils.js");
I am using the latest axois version 1.7.3

its working fine when i run the axois version 1.6.8
but after updating it to the latest version i dont know why its getting failed.

node version : v21.2.0 
yarn version: v1.22.19
npm version: v10.2.3

### Axios Version

1.7.3

### Browser

Chrome 

### Browser Version

127.0.6533.89

### Node.js Version

21.2.0

### OS

Window

### Additional Library Versions

```bash
React: V^18.2.0
```

**Comment by mellis481** [comment]
I am also seeing this in axios 1.7.4. Was not seeing it in 1.7.1. I'm only seeing it in Cypress component tests. We are also using Preact.

**Comment by mellis481** [comment]
@DigitalBrainJS Any reason to think something you included in #6518 could have caused this issue?

**Comment by mellis481** [comment]
@DigitalBrainJS Thoughts?

**Comment by jasonsaayman** [comment]
yeah, it's kind of well-known that there are some issues with testing. it seems... without good reproducible examples, we cannot really do much, so if you have a good example, please create a new issue with that example
